src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Default {
+    private double side;
+    private double firstLeg;
+    private double secondLeg;
```

Comment:
The variables 'side', 'firstLeg', and 'secondLeg' are not conceptually aligned with a rectangle's properties. Typically, a rectangle is defined by its 'length' and 'width'.